docs/source/reference/api-only.md

```diff
@@ -1,60 +1,47 @@
-exposes this functionality _primarily_ via a [REST API](rest).
-For convenience, JupyterHub also ships with a _basic_ web UI built using that REST API.
-The basic web UI enables users to click a button to quickly start and stop their servers,
-and it lets admins perform some basic user and server management tasks.
+Jupyterhub, as a service for deploying and managing Jupyter servers for users, exposes its functionality, _primarily_ through a [REST API](rest).
 
-The REST API has always provided additional functionality beyond what is available in the basic web UI.
-Similarly, we avoid implementing UI functionality that is also not available via the API.
-With JupyterHub 2.0, the basic web UI will **always** be composed using the REST API.
-In other words, no UI pages should rely on information not available via the REST API.
-Previously, some admin UI functionality could only be achieved via admin pages,
-such as paginated requests.
+For convenience, Jupyterhub also ships a basic web UI, built on the REST API. The basic web UI enables users to easily interact with their servers. E.g by clicking on the button to quickly start or stop their servers. It also enable admins to perform basic user and server mananagement tasks.
 
-## Limited UI customization via templates
+## The REST API
 
-The JupyterHub UI is customizable via extensible HTML [templates](templates),
-but this has some limited scope to what can be customized.
-Adding some content and messages to existing pages is well supported,
-but changing the page flow and what pages are available are beyond the scope of what is customizable.
+Previously, one could only achieve some admin UI functionalities through admin pages, such as paginated requests. The REST API provides additional functionality beyond what is available in the basic web UI. With Jupyterhub 2.0, the UI will always be composed from the REST API. This means that information on the pages will not be reliable, unless made available through the API.
 
-## Rich UI customization with REST API based apps
+## Why Rest API Only?
 
-Increasingly, JupyterHub is used purely as an API for managing Jupyter servers
-for other Jupyter-based applications that might want to present a different user experience.
-If you want a fully customized user experience,
-you can now disable the Hub UI and use your own pages together with the JupyterHub REST API
-to build your own web application to serve your users,
-relying on the Hub only as an API for managing users and servers.
+### Limited UI customization via templates
 
-One example of such an application is [BinderHub][], which powers https://mybinder.org,
-and motivates many of these changes.
+The JupyterHub UI can be customized through extensive HTML [templates](templates), but the scope at which it can be customized, is limited. It supports adding of content and messages to existing pages, but the page flow and available pages, cannot be customized.
```

Comment:
```suggestion
The JupyterHub UI can be customized through extensible HTML [templates](templates), but the scope at which it can be customized, is limited.
Custom templates support adding content and messages to existing pages, but the page flow and which pages are available cannot be customized.
```